src/bennu/devices/modules/comms/iec60870-5/module/Server.cpp

```diff
@@ -85,11 +99,56 @@ void Server::start(const std::string& endpoint, std::shared_ptr<Server> server,
     fflush(stdout);
 }
 
+DoublePointValue Server::convertBoolToDPValue(bool status)
+{
+    if (status == 0)
+        return IEC60870_DOUBLE_POINT_OFF;
+    else
+        return IEC60870_DOUBLE_POINT_ON;
+}
+
+DoublePointValue Server::convertIntToDPValue(int value)
+{
+    switch (value) 
+    {
+        case 0:
```

Comment:
Are cases 0 and 3 needed if the default case has same result? My C++ is a bit rusty.

Also, a `break` after `return` is redundant.

Reply:
The enums in 0 and 3 are different ("INTERMEDIATE" vs "INDETERMINATE")..though admittedly hard to see. To your point, case 0 and default are the same result, but I'd argue making case 0 explicit adds to readability/code understanding

Will remove the `break` statements